packages/labs/gen-wrapper-react/src/index.ts

```diff
@@ -14,27 +14,36 @@ import {
 } from '@lit-labs/analyzer/lib/model.js';
 import {javascript, FileTree} from '@lit-labs/gen-utils/lib/file-utils.js';
 
-export const generateReactWrapper = async (
-  analysis: Package
-): Promise<FileTree> => {
-  const litModules: LitModule[] = getLitModules(analysis);
-  if (litModules.length > 0) {
-    // Base the generated package folder name off the analyzed package folder
-    // name, not the npm package name, since that might have an npm org in it
-    const reactPkgFolder = packageNameToReactPackageName(
-      path.basename(analysis.rootDir)
-    );
-    return {
-      [reactPkgFolder]: {
-        '.gitignore': gitIgnoreTemplate(litModules),
-        'package.json': packageJsonTemplate(analysis.packageJson, litModules),
-        'tsconfig.json': tsconfigTemplate(),
-        ...wrapperFiles(analysis.packageJson, litModules),
-      },
-    };
-  } else {
-    throw new Error('No Lit components were found in this package.');
-  }
+export const getCommand = () => {
+  return {
+    name: 'react',
+    description: 'Generate React wrapper components from Lit elements',
+    kind: 'resolved',
+    async generate(options: {analysis: Package}): Promise<FileTree> {
```

Comment:
Could we keep a top-level export of `generate` for standalone usage, while also exporting a `getCommand()` for use in the CLI?

The idea is that other tools could call the generate function directly, so it's sort of weird if it has a very distinct Lit CLI shape. That would also address the need to make a generate-specific command type (comment above), since the command in `getCommand()` can just be called `run`.